lib/api/controllers/security.js

```diff
@@ -304,6 +304,7 @@ class SecurityController extends NativeController {
       description,
       { creatorId, refresh, apiKeyId });
 
+    this.kuzzle.log.info(`[SECURITY] User "${request.context.user}" applied action "${request.input.action}" on user "${userId}."`);
```

Comment:
Here `request.context.user` is an object, IMHO it's better to print only the ID
(This remark apply everywhere)